src/sql/workbench/parts/notebook/models/cell.ts

```diff
@@ -43,6 +43,7 @@ export class CellModel implements ICellModel {
 	private _onCellLoaded = new Emitter<string>();
 	private _loaded: boolean;
 	private _stdInVisible: boolean;
+	private _metadata: { language?: string; };
```

Comment:
Consider have this defined as a type alias in azdata so future updates don't need to keep this updated (and anywhere else this might be used)

Reply:
I hear you. I'll hold on this for now, but if and when we add more in this area will work on that.